packages/pwa/app/components/drawer-menu/index.jsx

```diff
@@ -285,21 +293,21 @@ const DrawerMenu = ({isOpen, onClose = noop, onLogoClick = noop, root}) => {
                                                 id: 'contactus',
                                                 name: intl.formatMessage({
                                                     id:
-                                                        'mobile_navigation.links.customer_support.contact_us',
+                                                        'drawer_menu.link.customer_support.contact_us',
                                                     defaultMessage: 'Contact Us'
                                                 })
                                             },
                                             {
                                                 id: 'shippingandreturns',
                                                 name: intl.formatMessage({
                                                     id:
-                                                        'mobile_navigation.links.customer_support.shipping_and_returns',
+                                                        'drawer_menu.link.customer_support.shipping_and_returns',
```

Comment:
The PR defined the id as having 3 parts, this one looks like it has 4. Are the rules not as simple as what you defined?

Reply:
I had a chat with @vmarta last week, we've agreed it is okay to add more hierarchical info to provide a better context on this id as long as we have three pieces of information on the id. @vmarta will put up a document to put the guideline to explain more on how to conduct a message-id for the pwa soon

Reply:
The 3 parts are simply guidelines. We can have more parts, if needed.

Or another ways to look at it is we're still having 3 _groups_, but each group can have more than 1 thing in it. In this case the 3rd group (the identifier group) has more of a hierarchy: `customer_support.shipping_and_returns`.